docs/content/latest/manage/backup-restore/point-in-time-recovery.md

```diff
@@ -154,4 +147,7 @@ Currently, you can recover from the following YCQL operations:
 Development for this feature is tracked in [issue 7120](https://github.com/yugabyte/yugabyte-db/issues/7120). Some forthcoming features include:
 
 * YCQL roles and permissions
-* Support for automatic tablet splitting
+
+### Common limitations
+
+* Currently, we don't support PITR in conjunction with xCluster replication.
```

Comment:
I'd file a GH issue for this, also explaining what works vs doesn't work. Data-only restore, for example, somewhat works, if you stop all writes and wait for replication lag to drop to 0. But metadata restore, ie: DDLs, would not work. It's important to make that distinction, as for some users, it might be enough to have data-only support here, much like for backups, we only have data-only support.

Speaking of which, let's also list the backup limitations here.

Reply:
Also, TRUNCATE is a limitation for both YCQL and YSQL.